config/config.go

```diff
@@ -233,6 +233,9 @@ type Adapter struct {
 	PlatformID string `mapstructure:"platform_id"`
 	AppID      string `mapstructure:"app_id"`
 	AppSecret  string `mapstructure:"app_secret"`
+
+	// needed for district m DMX
+	PublisherId string `mapstructure:"publisher_id"`
```

Comment:
Please consider using `ExtraAdapterInfo` for bidder specific configs. We added that recently to avoid many one-off  config entries.

Reply:
Yes, best to avoid one off additions to adapter configs.